kitsune/sumo/static/sumo/js/fxa-banner.js

```diff
@@ -0,0 +1,17 @@
+(function() {
+  'use strict';
+
+  var isFirefox = (/\s(Firefox)/.test(navigator.userAgent));
+
+  if (isFirefox) {
+    var banners = ['send', 'monitor', 'sync'];
+    var choice = Math.floor(Math.random() * 3);
```

Comment:
NIT: better to use `banners.length` here instead of `3` for future reasons, but not a big deal for this iteration obviously.